src/components/aboutPage/OurStory.jsx

```diff
@@ -8,7 +8,7 @@ export default function OurStory() {
         </div>
         <div className='mb-[50px] md:mb-[129px]'>
          <p className='max-w-[312px] md:max-w-[817px] text-center text-[16px] md:text-[24px] font-normal leading-[18.75px] md:leading-[28px]'>
-          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using Content here, content here, making it look like readable English.
+         At RefuBook, we understand the challenges that come with being a refugee and that is why we are dedicated to providing comprehensive support and resources to individuals who have been forced to flee their homes due to conflict, persecution, or other life-threatening circumstances. We believe that every refugee deserves the opportunity to thrive in their new surroundings. Through our platform, we connect refugees with resources and support, such as housing, medical care, education, and employment. We also work to advocate for the rights and needs of refugees on all levels, locally, nationally and internationally. But we can not do it alone, with your help, by volunteering, donating, spreading awareness, or simply welcoming new refugees in your community, you can make a meaningful difference in the lives of refugees. Join us in creating a more inclusive and welcoming world for all.
```

Comment:
you should give some line-height between text and also divide text content with <br/>  once certain context is finished.